src/fluree/server/system.clj

```diff
@@ -55,6 +56,11 @@
                       :watcher       (ig/ref :fluree/watcher)
                       :subscriptions (ig/ref :fluree/subscriptions)}}))
 
+(defmethod ig/expand-key ::config/sid-migration
+  [_ sid-migration]
+  {:fluree/sid-migration {:sid-migration sid-migration
+                          :conn (ig/ref :fluree/connection)}})
```

Comment:
This is minor, but the resulting config map would be easier to deal with if it was flattened didn't have this extra `::sid-migragion` key inside of it.  I think this would be better:

```suggestion
(defmethod ig/expand-key ::config/sid-migration
  [_ config]
  (let [config* (assoc config :conn (ig/ref :fluree/connection))]
    {:fluree/sid-migration config*}))
```